contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -7,6 +7,7 @@ cluster:
     data-path: "/datastorage"
     qos-switch: "{{ env["cfg"]["qos-switch"] | default('false') }}"
     docker-data-root: "{{ env['cfg']['docker_data_root'] | default('/mnt/docker') }}"
+    marketplace: "true"
```

Comment:
Maybe `marketplace: "false"`? 

Or if we decide to turn on marketplace by default, better to notify all PAI devs.

Reply:
Done, set to "false".